test/reader.test.js

```diff
@@ -424,3 +424,63 @@ test('multiple entrypoints in same feed supported', async () => {
     expect(spy).toMatchSnapshot();
     expect(spy).toHaveBeenCalledTimes(4);
 });
+
+test('should replace NODE_ENV with default of "development"', async () => {
+    expect.hasAssertions();
+    const env = process.env.NODE_ENV;
+    delete process.env.NODE_ENV;
+    const result = await bundleJS([
+        [
+            {
+                id: 'a',
+                entry: true,
+                file: './asdas.js',
+                source: 'console.log(process.env.NODE_ENV)',
+                deps: {},
+            },
+        ],
+    ]);
+    expect(result).toMatch('development');
+    expect(prettier.format(result)).toMatchSnapshot();
+    process.env.NODE_ENV = env;
```

Comment:
cleanup will only run if the assertions doesn't fail, but maybe that's ok

Reply:
Hmm true. Yea gonna go with "should be fine" for now.